bindings/python/setup.py

```diff
@@ -245,7 +245,7 @@ def join_all(src, files):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 3',
     ],
-    requires=['ctypes'],
+    requires=['ctypes', 'typing; python_version<"3.5"'],
```

Comment:
By doing this, you make the Unicorn module now depend on typing, where previously the user didn't need to install any other packages in order to use Unicorn, other than the Unicorn package itself. `ctypes` is a built-in in Python 2.7, so the existing requirement is not strictly necessary.